cohere/response.py

```diff
@@ -66,5 +66,35 @@ def __repr__(self) -> str:
         output = f'cohere.{type(self).__name__} {{\n{contents}}}'
         return output
 
-    def feedback(self, feedback: str, accepted: bool) -> Feedback:
-        return self.client.feedback(id=self.id, feedback=feedback, accepted=accepted)
+    def feedback(self, good_response: bool, desired_response: str = "", feedback: str = "") -> Feedback:
```

Comment:
a bit confusing that the function is named feedback, and so is the param. Maybe something like this?
```suggestion
    def feedback(self, good_response: bool, desired_response: str = "", additional_feedback: str = "") -> Feedback:
```